platformio.ini

```diff
@@ -435,19 +433,18 @@ board_build.partitions = ${esp32.default_partitions}
 [env:esp32dev_V4]
 board = esp32dev
 platform = ${esp32_idf_V4.platform}
-platform_packages = ${esp32_idf_V4.platform_packages}
 build_unflags = ${common.build_unflags}
 build_flags = ${common.build_flags} ${esp32_idf_V4.build_flags} -D WLED_RELEASE_NAME=\"ESP32_V4\" #-D WLED_DISABLE_BROWNOUT_DET
   ${esp32.AR_build_flags}
 lib_deps = ${esp32_idf_V4.lib_deps}
   ${esp32.AR_lib_deps}
 monitor_filters = esp32_exception_decoder
 board_build.partitions = ${esp32.default_partitions}
+board_build.flash_mode = qio ;; TODO: is this the correct flash mode?
```

Comment:
Is this the correct value? Adding fixed the missing sdkconfig.h error

Reply:
I don't remember when the use of faulty bootloaders for mode QIO was fixed. Core 2.0.9 is really old.

Reply:
I have updated flash_mode to dio now.

As for 2.0.9 we need to stay on 2.x due the breaking changes in 3.x that affect many of the libraries we use.

We have found various bugs with later releases of 2.0.x so hence why @softhack007 wants to stick with 2.0.9 for now